apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -3,7 +3,7 @@
 import { CollapseIcon, Icon } from "@/components/icon";
 import clsx from "clsx";
```

Comment:
![image](https://github.com/user-attachments/assets/79f88c91-6872-4e22-ae45-5d1d5b35b76c)
I would only document Inline as standalone is the default. We usually don't do this but it's worth having a dedicated section since inline should only be used when contained. I would display a real world scenario, maybe a Disclosure inside a Card.

Reply:
@vicky-comeau Rethinking about it I would still call the section Variant to stay in line with other components.But would make sure have a dedicated section explaining the why of Inline.